public/app/features/scenes/components/VizPanel/VizPanelRenderer.tsx

```diff
@@ -14,14 +14,16 @@ import { SceneDragHandle } from '../SceneDragHandle';
 import { VizPanel } from './VizPanel';
 
 export function VizPanelRenderer({ model }: SceneComponentProps<VizPanel>) {
-  const { title, options, fieldConfig, pluginId, pluginLoadError, $data, ...state } = model.useState();
+  const { title, options, fieldConfig, pluginId, pluginLoadError, $data, layout: layoutOptions } = model.useState();
   const [ref, { width, height }] = useMeasure();
   const plugin = model.getPlugin();
   const { data } = sceneGraph.getData(model).useState();
-  const layout = sceneGraph.getLayout(model);
+  const parentLayout = sceneGraph.getLayout(model);
```

Comment:
@ivanortegaalba @dprokop yea I agree, I struggled with the nameing of the options prop alot (the size prop, which is renamed to layout in this PR).

A more clear name would be:
* layoutOptions   (but it's so long and it's going to be a very common property) 
* layoutStyle    (feel that just style is is going to be confusing with look / design)
* style
* layout
* layoutChild (a bit long and complicated) 
* gridPos  (only fits great for grid layout, and does not fit the other props)
* layoutOpts  (shorter but not as clear as full layoutOptions) 

Non feel 100% right, what's your top picks?

Reply:
I think either `layout` or `placement` would work well. To avoid confusion of `sceneGraph.getLayout` returning scene object, I think we can consider renaming it to `sceneGraph.getLayoutObject`.

Reply:
hm.. like placement